CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
    * CHANGED: Replace unstable c++ geos API with a mix of geos' c api and boost::geometry for admin building [#3683](https://github.com/valhalla/valhalla/pull/3683)
    * ADDED: optional write-access to traffic extract from GraphReader [#3876](https://github.com/valhalla/valhalla/pull/3876)
    * UPDATED: locales from Transifex [#3879](https://github.com/valhalla/valhalla/pull/3879)
+   * FIXED: Build most of Baldr with -Werror [#3885](https://github.com/valhalla/valhalla/pull/3885)
```

Comment:
bug fixes go at the bottom of the "bug fix" subsection. i would probably call this a change though not a bug, would you mind changing this to changed?
```suggestion
   * CHANGED: Build most of Baldr with -Werror [#3885](https://github.com/valhalla/valhalla/pull/3885)
```

Reply:
Yes no problem